tests/unit/test_ocelots_pandas_utils.py

```
@@ -0,0 +1,114 @@
+
+    parsed = json.loads(result)
+    assert parsed == nested_value
+
+
+def test_safe_convert_to_string_regular_values():
+    assert safe_convert_to_string("hello") == "hello"
+
+    assert safe_convert_to_string(42) == "42"
+    assert safe_convert_to_string(3.14) == "3.14"
+
+    assert safe_convert_to_string(True) == "True"
```

Ruff (0.14.3)

Boolean positional value in function call

(FBT003)